packages/fixie/tsconfig.json

```diff
@@ -9,10 +9,11 @@
     "moduleResolution": "node16",
     "jsx": "react",
     "esModuleInterop": true,
-    "outDir": ".",
+    "outDir": "./dist",
+    "rootDir": "./src",
     "declaration": true,
     "lib": ["dom", "dom.iterable", "ES2022"]
   },
-  "include": ["src/*", "jest.config.ts", ".eslintrc.cjs", "index.ts", "web.ts"],
+  "include": ["src/**/*", "jest.config.ts", ".eslintrc.cjs", "index.ts"],
```

Comment:
I'm pretty sure you're breaking into jail w.r.t. ESM/CJS/TS because Next.js apps default to CJS -- at the very least you should do this as part of a different change